### ISSUE TITLE
[WebProfilerBundle][2.6]Exception on the Logger Tab

### ISSUE THREAD

**Issue by jewome62** [symfony/symfony#14695]
I have an Deprecated Calls in my web profiler toolbar. When i click on them, I have this exception
``
Variable "id" does not exist in @WebProfiler/Collector/logger.html.twig at line 117
 500 Internal Server Error - Twig_Error_Runtime 
``
URL :
app_dev.php/_profiler/a7a649?panel=logger
This is only the logger tab

My dependancies
```
composer show -i
behat/transliterator                 v1.0.1             String transliterator
doctrine/annotations                 v1.2.4             Docblock Annotations Parser
doctrine/cache                       v1.4.1             Caching library offering an object-oriented API for many cache backends
doctrine/collections                 v1.3.0             Collections Abstraction library
doctrine/common                      v2.5.0             Common Library for Doctrine projects
doctrine/data-fixtures               v1.1.1             Data Fixtures for all Doctrine Object Managers
doctrine/dbal                        v2.4.4             Database Abstraction Layer
doctrine/doctrine-bundle             v1.4.0             Symfony DoctrineBundle
doctrine/doctrine-cache-bundle       v1.0.1             Symfony2 Bundle for Doctrine Cache
doctrine/doctrine-fixtures-bundle    v2.2.0             Symfony DoctrineFixturesBundle
doctrine/inflector                   v1.0.1             Common String Manipulations with regard to casing and singular/plural rules.
doctrine/lexer                       v1.0.1             Base library for a lexer that can be used in Top-Down, Recursive Descent Parsers.
doctrine/orm                         v2.4.7             Object-Relational-Mapper for PHP
friendsofsymfony/jsrouting-bundle    1.5.4              A pretty nice way to expose your Symfony2 routing to client applications.
gedmo/doctrine-extensions            v2.3.12            Doctrine2 behavioral extensions
goodby/csv                           1.2.0              CSV import/export library
gregwar/cache                        v1.0.10            A lightweight file-system cache system
gregwar/image                        v2.0.19            Image handling
gregwar/image-bundle                 dev-master 5953a49 Image handling bundle
guzzlehttp/guzzle                    5.2.0              Guzzle is a PHP HTTP client library and framework for building RESTful web service clients
guzzlehttp/ringphp                   1.0.7              Provides a simple API and specification that abstracts away the details of HTTP into a single PHP function.
guzzlehttp/streams                   3.0.0              Provides a simple abstraction over streams of data
incenteev/composer-parameter-handler v2.1.0             Composer script handling your ignored parameter file
jdorn/sql-formatter                  v1.2.17            a PHP SQL highlighting library
kriswallsmith/assetic                v1.2.1             Asset Management for PHP
mobiledetect/mobiledetectlib         2.8.13             Mobile_Detect is a lightweight PHP class for detecting mobile devices. It uses the User-Agent string combined with specific HTTP headers to detect the mobile environment.
monolog/monolog                      1.13.1             Sends your logs to files, sockets, inboxes, databases and various web services
phpoffice/phpexcel                   1.8.1              PHPExcel - OpenXML - Read, Create and Write Spreadsheet documents in PHP - Spreadsheet engine
psr/log                              1.0.0              Common interface for logging libraries
react/promise                        v2.2.0             A lightweight implementation of CommonJS Promises/A for PHP
sensio/distribution-bundle           v3.0.24            Base bundle for Symfony Distributions
sensio/framework-extra-bundle        v3.0.7             This bundle provides a way to configure your controllers with annotations
sensio/generator-bundle              v2.5.3             This bundle generates code for you
sensiolabs/security-checker          v2.0.2             A security checker for your composer.lock
stfalcon/tinymce-bundle              dev-master 758c199 This Bundle integrates TinyMCE WYSIWYG editor into a Symfony2 project.
stof/doctrine-extensions-bundle      v1.1.0             Integration of the gedmo/doctrine-extensions with Symfony2
suncat/mobile-detect-bundle          v0.10.1            Symfony2 bundle for detect mobile devices, managing mobile view types, redirect to mobile version.
swiftmailer/swiftmailer              v5.4.0             Swiftmailer, free feature-rich PHP mailer
symfony/assetic-bundle               v2.6.1             Integrates Assetic into Symfony2
symfony/monolog-bundle               v2.7.1             Symfony MonologBundle
symfony/swiftmailer-bundle           v2.3.8             Symfony SwiftmailerBundle
symfony/symfony                      v2.6.7             The Symfony PHP framework
twig/extensions                      v1.2.0             Common additional features for Twig that do not directly belong in core
twig/twig                            v1.18.1            Twig, the flexible, fast, and secure template language for PHP
willdurand/jsonp-callback-validator  v1.1.0             JSONP callback validator.
```

**Comment by stof** [comment]
Thanks for the report.

**Comment by stof** [comment]
This is now fixed in the 2.6 branch
